### PR TITLE
Image Worker fails on CF Redirects

### PR DESCRIPTION
**Pull request by vikiival**
The problem:

Looks like the gateway cannot correcly process redirects to CF gateways

- :bug: failed to fetch on KodaDot
- :bug: redirect fail on CF_Gateway

Now

Screenshot 2023-07-31 at 14 00 50

This PR

Screenshot 2023-07-31 at 14 00 09

### CODE REVIEW

**Comment by preschian** [comment]
is this using v2?
https://squid.subsquid.io/speck/v/v2/graphql

null on v1
image

File: image/src/index.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ app.all('/ipfs/*', async (c) => {
       }
 
       // fallback to cf-ipfs
-      return Response.redirect(`${c.env.CLOUDFLARE_GATEWAY}/ipfs/${path}`, 302);
+      return c.redirect(`${c.env.CLOUDFLARE_GATEWAY}/ipfs/${path}`, 302);
```
Comment (preschian) [inline image/src/index.ts:76]: Are we sure this is the fix? There are total of 5 lines using `Response.redirect`

**Review by preschian — APPROVED** [review]
need to change another `Response.redirect`, otherwise let's test it on beta first

**Comment by vikiival** [comment]
Like for me it's unusable without this change.
Still throws some CORS errors so I would suggest fix the service to the top-notch state

**Pull request merged**